src/shared/form/ErrorBox.js

```diff
@@ -15,6 +15,11 @@ class ErrorBox extends React.Component {
 
```

Comment:
this file still has a reference to CaseStudy - I need to use it in SellerRegistration